ludwig/data/preprocessing.py

```diff
@@ -1282,7 +1367,7 @@ def get_split(
             if backend.df_engine.partitioned:
                 # This approach is very inefficient for partitioned backends, which
                 # can split by partition
-                return None
+                return
```

Comment:
This suggests that we are not creating a split at all in this case?

Reply:
So what happens here is the split for dask dataframes happens at a higher up level since splitting dask dataframes in the get_split() function is very slow because you have to call set_index() which is an expensive dask method. So i decided to pull the function out of the build_dataset function altogether and just operate on the training split of the dataset after its been defined in the _preprocess_df_for_training and _preprocess_file_for_training functions. The reason there is a change here at all was because I had made a splitting function for dask dataframes initially, but then deleted it all after I realized how slow it would be but forgot to put back the None in the return. I can add it back in now.